### PR TITLE
replace `async defer` by `defer` for <script> + misc linter

### PR DESCRIPTION
**Pull request by divinerites**
Some trailing space where removed by my editor (VSC) to comply for md lint

### CODE REVIEW

**Comment by divinerites** [comment]
And also https://github.com/plausible/analytics/pull/1107

File: docs/docusaurus-integration.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This is how [we've integrated Plausible](https://github.com/plausible/docs/commi
 
 ```
 module.exports = {
-  scripts: [{src: 'https://plausible.io/js/plausible.js', async: true, defer: true, 'data-domain': 'yourdomain.com'}],
+  scripts: [{src: 'https://plausible.io/js/plausible.js', async: false, defer: true, 'data-domain': 'yourdomain.com'}],
```
Comment (ukutaht) [inline docs/docusaurus-integration.md:17]: Just leave async out in this line.

**Review by ukutaht — CHANGES_REQUESTED** [review]
Looks, good. I would just make one change in the docusaurus config, see comment below.

**Comment by ukutaht** [comment]
Thanks!

**Pull request merged**